main.css

```diff
@@ -85,6 +93,10 @@
     padding: 0;
     text-decoration: none;
     display: block;
+    padding: 3px 5px 3px 16px;
+    }
+    .dropdown-content a:hover {
+        background: rgba(58, 137, 225, 0.6);
```

Comment:
This is the blue hover color on the links in the dropdown. You could change this to orange if you want; or make it change the text color and not the background if you like that better.